.ci/check

```diff
@@ -6,30 +6,30 @@ set -e
 # SOURCE_PATH - path to component repository root directory.
 
 if [[ -z "${SOURCE_PATH}" ]]; then
-  export SOURCE_PATH="$(readlink -f "$(dirname ${0})/..")"
+  SOURCE_PATH="$(readlink -f "$(dirname "${0}")/..")"
+  export SOURCE_PATH
 else
-  export SOURCE_PATH="$(readlink -f ${SOURCE_PATH})"
+  SOURCE_PATH="$(readlink -f "${SOURCE_PATH}")"
+  export SOURCE_PATH
 fi
 
 # Install Golint.
-if ! which golint 1>/dev/null; then
-  echo -n "Installing golint... "
-  go install golang.org/x/lint/golint@v0.0.0-20210508222113-6edffad5e616
-  echo "done."
+if ! which golangci-lint 1>/dev/null; then
+  echo -n "Installing golangci-lint... "
+  curl -sSfL https://raw.githubusercontent.com/golangci/golangci-lint/master/install.sh | sh -s -- -b "$(go env GOPATH)"/bin v1.56.0
+  golangci-lint --version
```

Comment:
Maybe we could move the `golangci-lint --version` line out of the if statement, so that the version of `golangci-lint` is always shown in the output. This way we could easily spot issues due to old versions of `golangci-lint` which are otherwise hard to spot.